server/routes/getTokenForum.ts

```diff
@@ -31,7 +31,7 @@ const getTokenForum = async (
           where: { id: token.id },
           defaults: {
             active: true,
-            network: token.id,
+            network: 'erc20',
```

Comment:
Can we turn `'erc20'` into a constant?

Reply:
Unsure what enum we would use here, although we have one on the client.